blog/_posts/2017-08-03-Codeaholics,-the-first-hangover.md

```diff
@@ -0,0 +1,78 @@
+
+![image](/img/blog/2017/codeaholics_working.jpg)
+<div class="image-credits">Team Codeaholics - Working time!</div>
+
+### __RGSoC, let's apply!__
+
+Jona's and Xheni's latest involvement into open source is [Nextcloud](https://nextcloud.com/), a suite of client-server software for creating and using file hosting service, where their contributions consisted mostly on promoting Nextcloud and translating it in their local language (shqip).
+
+At OSCAL (third edition) they met Jan-Christoph Borchardt and had the chance to talk with him. After some months, they saw his tweet about a
+dedicated scholarship program that aims to foster diversity in Open Source since 2013, called "Rails Girls Summer of Code". Both of them were
+really enthusiast and happy to hear about such a great opportunity that could help girls develop their skills and gain new exciting experiences. After posting this on the [Open Labs forum](https://forum.openlabs.cc/) to share it with all the girls, they even organized an "unofficial" meeting with all the interested girls to help each other with the process. It was the first time that 4 teams from Albania applied to be part of RGSoC!
+
+### __Why Nextcloud?__
+
+After picking each other as teammates, Jona and Xheni had to decide about the project they wanted to spend all their summer working on. Wait, what about the name of the team?
+Hmmm let's say they needed only a couple of hours to find the appropriate name for them. And it was not very difficult, because both of them are __CODEAHOLICS__. Yes, I mean it! They stay awake all night long and they like to sleep in the morning because for them is way easier to work when the place is very quiet, people are sleeping and the only thing that they can hear is their favorite music. Okay now let's get back to the topic. After scouting all the projects listed, the first one that was deemed a perfect fit for them was Nextcloud. Why? Because they were already using it and they were familiar (not a lot) with the community. Both of them believe that when you use something and you like it, it's very easy for you to start promoting and working on it, in this way you will always do something that you love and are not forced to. They decided to work on the contacts app and documentation part during the summer.
+After finishing the application all the albanian teams were "stressed" waiting about the final result. Who could imagine that we would be one of the lucky teams, being the first albanian team part of RGSoC! Imagine their happy faces when they saw the approval email, __yaaay__ :D
+
+![image](/img/blog/2017/codeaholics_2.jpg)
```

Comment:
could you please compress/resize `codeaholics_2.jpg` and `codeaholics_team.png` so that they are under 2MB? The files are very big, which is an issue for people cloning the repo for the first time :)